.github/workflows/pytest_matrix.yml

```diff
@@ -93,6 +93,15 @@ jobs:
           poetry run coverage html -d htmlcov
           poetry run coverage xml -o htmlcov/coverage.xml
 
+      - name: Publish Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v2
+        if: always() && !cancelled() && steps.changes.outputs.src == 'true'
+        continue-on-error: true
+        with:
+          check_name: "PyTest Results (Matrix, Python ${{ matrix.python-version }}, ${{ matrix.os }})"
```

Comment:
Let's modify the `if` so that this only runs on Python 3.11.

And then we can rename simply as: `check_name: PyTest Results (Full)`